packages/vite/src/client.ts

```diff
@@ -68,7 +69,10 @@ export async function buildClient (ctx: ViteBuildContext) {
       runtimePathsPlugin({
         sourcemap: ctx.nuxt.options.sourcemap.client
       }),
-      viteNodePlugin(ctx)
+      viteNodePlugin(ctx),
+      chunkErrorPlugin({
```

Comment:
Should we also add the plugin conditionally according to `reloadOnRouteChunkError` option ?

Reply:
Good thought! I was thinking with it false they might handle it themselves. Maybe support instead via configuring it as either `true | false | 'manual'` (the last one raises events but doesn't handle)?

Reply:
> the last one raises events but doesn't handle

Sounds great!

Reply:
~> refactored to `emitRouteChunkError` with `true | false | 'reload'` signature.

Reply:
This also means we can potentially merge in a patch release for testing. 👀